### PR TITLE
migrate to proj 7.0.0

### PR DESCRIPTION
**Pull request by ocefpaf**
Migrate from 6.3.1 to 7.0.0.

### CODE REVIEW

**Comment by beckermr** [comment]
Needs a version bump to post tomorrow, so april 8.

File: recipe/migrations/proj700.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  # This determines the increment to the build number when the 
+  # migration runs. 
+  # Change this to zero if the new pin increases the number of builds
+  build_number:
+    1
+
+# The name of the feedstock you wish to migrate with dashes replaced by
+# underscores
+proj:
+  - 7.0.0    # new version to build against
```
Comment (isuruf) [inline recipe/migrations/proj700.yaml:23]:
```suggestion
  - 7
```

Comment (CJ-Wright) [reply recipe/migrations/proj700.yaml:23]: Actually it looks like it is pinned down to micro https://github.com/conda-forge/proj.4-feedstock/blob/master/recipe/meta.yaml#L17

**Comment by CJ-Wright** [comment]
I'm surprised the bot didn't pick this one up.

File: recipe/migrations/proj700.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/migrations/proj700.yaml:23]: Yep. Proj has a terrible history of ABI breakages. See https://abi-laboratory.pro/?view=timeline&l=proj

**Comment by ocefpaf** [comment]
Ping!

**Comment by ocefpaf** [comment]
@CJ-Wright can we merge https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/534 first to avoid creating two packages?

**Pull request merged**